shared/firebase-common.ts

```diff
@@ -1,13 +1,9 @@
 import * as firebase from 'firebase-admin';
 
-const config = {
-  // TODO(josephperrott): Set up config via ENV variables.
-};
-
 /** The initialized Firebase app instance */
-let firebaseApp: firebase.app.App;
+export const firebaseInst = firebase.initializeApp();
+/** The initialized firestore instance */
+export const firestoreInst = firebaseInst.firestore();
```

Comment:
Nit: Not a fan of `Inst`, why not just `firestore` or `firestoreInstance`. I think it would be fine to just call this `firestore` if you want to keep it short.